cmd/osbuild-service-maintenance/aws.go

```diff
@@ -82,5 +83,22 @@ func AWSCleanup(maxConcurrentRequests int, dryRun bool, accessKeyID, accessKey s
 		wg.Wait()
 	}
 
+	// Terminate leftover secure instances
+	reservations, err := a.DescribeInstancesByTag("parent", "i-*")
+	if err != nil {
+		return fmt.Errorf("Unable to describe instances by tag %w", err)
+	}
+	var instanceIDs []string
+	for _, res := range reservations {
+		for _, i := range res.Instances {
+			if i.LaunchTime.Before(time.Now().Add(-time.Hour * 2)) {
```

Comment:
should we make this configurable?

Reply:
Well currently the timeout would have to change together with the timeouts in internal/executor, so I think it's fine.